package.json

```diff
@@ -9,7 +9,7 @@
     "fs": false,
     "stream": "readable-stream",
     "http": "stream-http",
-    "ipfs-api": false,
+    "ipfs-http-client": false,
```

Comment:
do we need this ?
```suggestion
    
```

Reply:
I would think not, but lets deal with it separately...

Reply:
ok